packages/e2e/features/ui/components/storage/storage-browser/error-boundary.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Storage Browser uses error boundry
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+
+  @react
+  Scenario: Attempt to access forbidden folder
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I see the "Sign in" button
+    When I click the "Sign in" button
+    When I click the "s3://fileuploaderbucket121216-dev/forbidden/*" button
+    Then I see "no identity-based policy allows the s3:ListBucket action"
+  
+  
+  @react
+  Scenario: Attempt to add folder without write access
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I see the "Sign in" button
+    When I click the "Sign in" button
+    When I click the "s3://fileuploaderbucket121216-dev/forbidden/*" button
+    Then I see the "Actions" button
+    When I click the "Actions" button
+    Then I see the "Create Folder" menuitem
+    When I click the "Create Folder" menuitem
+    Then I see "Enter folder name:"
+    Then I see the "Create Folder" button
+    Then the "Create Folder" button is disabled 
+    When I type a new "Enter folder name:" with value "Blackberry"
+    Then I see the "Create Folder" button
+    When I click the "Create Folder" button
+    Then I see "no identity-based policy allows the s3:PutObject action"
```

Comment:
I was not sure what the best way of identifying the error boundary would be. Here, I am ensuring that the text within the error appears on the page. I'm open to suggestions on a more effective method of testing this error boundary would be